### ISSUE TITLE
Unable to copy data with keyboard only

### ISSUE THREAD

**Issue by connlim** [connlim/pe#8]
The app allows users to copy data by right-clicking, but not with only the keyboard. 

While it is a useful feature, the project constraints call for typing being the preferred method of interaction. So there should be a command to let users optionally copy data without using the mouse. 

As stated on the [project requirements website](https://nus-cs2103-ay2223s1.github.io/website/admin/tp-constraints.html#recommendation-cli-first):

> it simply means the input to the app should be a text command, which can even be entered using a GUI (similar to how AB3 does it).

**Comment by nus-pe-script** [comment]
# Team's Response
There was a decision to not include the CLI-version for now as it is more of an optional feature for the users, which is not part of the list of features that are documented in both the user guide and developer guide. An addition of a CLI-based command would require a lot of modification and hence the feature was not included.
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.NotInScope`]

- [x] I disagree

**Reason for disagreement:** Addition of the feature would be in scope as this is part of the project constraints to have a fully keyboard driven GUI. Addition is also not difficult if GUI copy and pasting already works.

-------------------
